glide.yaml

```diff
@@ -72,10 +73,12 @@ import:
 # master
 - package: github.com/onsi/ginkgo
   repo:    https://github.com/openshift/onsi-ginkgo.git
-  version: release-v1.2.0
+  # the commit below points to release-v1.2.0 branch
+  version: 6e93088a6ab736ec6a12d10ef99962f8d9e435ee
 - package: github.com/containers/image
   repo:    https://github.com/openshift/containers-image.git
-  version: openshift-3.8
+  # the commit below points to openshift-3.8 branch
```

Comment:
missing the reason why we pin it to a commit in the first place, also everywhere below.